conformance/echo-basic/echo-basic.go

```diff
@@ -130,6 +133,11 @@ func main() {
 	}
 }
 
+func wsHandler(ws *websocket.Conn) {
+	fmt.Println("established websocket connection", ws.RemoteAddr())
+	io.Copy(ws, ws)
```

Comment:
this is a one liner to ws.Read() and ws.Write(..) ?

Reply:
yeah websocket.Conn is an `io.ReadWriter` so it's reading frames and writing them back

Got it from https://pkg.go.dev/golang.org/x/net/websocket#example-Handler

Reply:
cool, comment here would be great

Reply:
+1